plugins/inputs/internet_speed/internet_speed.go

```diff
@@ -12,13 +12,23 @@ import (
 // InternetSpeed is used to store configuration values.
 type InternetSpeed struct {
 	EnableFileDownload bool            `toml:"enable_file_download"`
+	Offset             string          `toml:"offset"`
+	Cache              bool            `toml:"cache"`
 	Log                telegraf.Logger `toml:"-"`
 }
 
 const sampleConfig = `
   ## Sets if runs file download test
-  ## Default: false  
-  enable_file_download = false
+  ## Default: false
+  enable_file_download = true
+
+  ## Time to sleep before running the speed test
+  ## Default: ""
+  offset = "1m"
+
+  ## Caches the closest server location
+  ## Default: false
+  cache true
```

Comment:
Please adhere to the [sample-config style](https://github.com/influxdata/telegraf/blob/master/docs/developers/SAMPLE_CONFIG.md#style)
```suggestion
  # enable_file_download = false

  ## Time to sleep before running the speed test
  # offset = "0s"

  ## Caches the closest server location
  # cache = false
```
Furthermore, don't change existing values (`enable_file_download`). Also fixed a syntax error for `cache`.

Reply:
Please synchronize this section with the README.md content.